### PR TITLE
Getting Started With MariaDB Using Docker and Node.js

### PR DESCRIPTION
**Pull request by ephnjor2021**
### Article checklist

**ATTENTION: In your PR - add the ISSUE (topic form) # that this PR corresponds to. You can do this by also adding a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.**
      
      
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**If you have not already please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve our overall technical writing.**

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by ephnjor2021** [comment]
Solves issue #2244

**Comment by hectorkambow** [comment]
Thank you v much for your patience as we worked through our Repo re-structuring process. 

The purpose behind this restructure - was to better allow us to handle more volume and allow students to publish more articles and get more involved with the community (with open-source contribution and suggestions). 

With most of the changes coming to completion - we will be sure to get your article published asap. 

Thank you again - and we are looking forward to more amazing articles.

File: articles/getting-started-with-mariadb-using-docker-and-nodejs/index.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+The First will be to create a new reusable [module](https://nodejs.org/api/modules.html) file by the name `db.js`.
+
+The `db.js` module will utilize the MariaDB Node.js connector to enable our application to connect to and communicate with MariaDB.
+
+The code for the `db.js` will be as follows:
+
+```JavaScript
+// here we import the mariadb
+var mariadb = require('mariadb');
```
Comment (adrianmurage) [inline articles/getting-started-with-mariadb-using-docker-and-nodejs/index.md:97]: Preferably use `const` as this variable isn't going to be changing in the duration of the app. 

```suggestion
const mariadb = require('mariadb');
```

**Review by adrianmurage — APPROVED** [review]
@ephnjor2021 
Great article! Looking forward to seeing more of your work =)
I pointed out a minor recommendation.

**Comment by adrianmurage** [comment]
@hectorkambow 
This article is ready for final review.

**Comment by louisefindlay23** [comment]
@ephnjor2021, can you update your forked repo with the latest commits on the main Section Eng-Ed repo? There's been some structural changes in order to improve the EngEd programme and your PR needs the latest version so I can preview your article to complete the final review and publish it.

Here's some [instructions](https://www.earthdatascience.org/courses/intro-to-earth-data-science/git-github/github-collaboration/update-github-repositories-with-changes-by-others/) on how to do this via the command line or via the GitHub website. If this is the first time you've updated your forked repo, I'd recommend do this via the GitHub website as it's easier. Just open a new PR, click compare across forks and do a reverse PR (merging EngEd master into your forked master) and the merge your forked master branch.

**Comment by ephnjor2021** [comment]
@louisefindlay23 Okay, looking at it now

**Comment by ephnjor2021** [comment]
@louisefindlay23 I have done reverse PR, kindly look at it

**Review by louisefindlay23 — APPROVED** [review]
@ephnjor2021, thanks for updating your forked repo. Final review is complete and your article is published. Congrats.

**Comment by ephnjor2021** [comment]
@louisefindlay23 Thank you

**Pull request merged**